exasol/ds/sandbox/lib/dicts.py

```diff
@@ -0,0 +1,43 @@
+from typing import Any
+
+
+class DictAccessor:
+    """
+    Access values in a nested dictionary via key sequences.
+    """
+
+    def __init__(self, raw: dict[str, Any], prefixes: list[str]|None = None):
+        self._raw = raw
+        self.prefixes = prefixes or []
+
+    def _nested(self, *keys: str) -> Any:
+        current = self._raw
+        for key in keys:
+            if key not in current:
+                return None
+            current = current[key]  # type: ignore
+        return current
+
+    def get(self, *keys: str) -> Any:
+        return self._nested(*self.prefixes, *keys)
+
+    def as_dict(
+        self,
+        spec: dict[str, tuple[str, ...]],
+    ) -> dict[str, Any]:
+        """
+        Parameter ``spec`` maps output entries to a tuple of access keys.
+        Each output entry is a string, while the access keys is tuple of keys
+        to access the nested entries.
+
+        spec = { "output-entry": ("key-1", "key-2") }
```

Comment:
Here would be a complete example useful, means with all inputs and the expected output.

Reply:
I added an example.

Please also take a look at the unit tests.